app/models/work_type.rb

```diff
@@ -134,9 +134,9 @@ def self.all
     [
       new(id: 'data', label: 'Data', icon: 'chart-bar', subtypes: DATA_TYPES),
       new(id: 'image', label: 'Image', icon: 'images', subtypes: IMAGE_TYPES),
-      new(id: 'mixed material', label: 'Multimedia', icon: 'play', subtypes: MIXED_TYPES),
+      new(id: 'mixed materials', label: 'Mixed Materials', icon: 'play', subtypes: MIXED_TYPES),
       new(id: 'other', label: 'Other', icon: 'archive', subtypes: []),
-      new(id: 'software, multimedia', label: 'Software or Code', icon: 'mouse', subtypes: SOFTWARE_TYPES),
+      new(id: 'software, mixed materials', label: 'Software or Code', icon: 'mouse', subtypes: SOFTWARE_TYPES),
```

Comment:
@jcoyne reading the associated ticket and looking at https://docs.google.com/document/d/1tEdGP3tk-IG9ICsaN2ilMhn0UWu1pM7-Y6QFqMtVkoM/edit, it's not clear to me whether this change (to software) is included in what we want. I don't see any guidance related to the IDs in this list in the documentation, and I don't see any information in the doc that would suggest we need anything other than, say, `software` here. Pls advise?

Reply:
This is a mods resource type, and it shouldn't change: https://github.com/sul-dlss-labs/cocina-descriptive-metadata/blob/master/h2_cocina_mappings/h2_to_cocina_form.txt#L162

Reply:
Restored `multimedia` for **Software or Code** in latest commit